src/fortran_routing/mc_pylink_v00/MC_singleSeg_singleTS/mc_sseg_stime_NOLOOP_demo.py

```diff
@@ -62,8 +131,8 @@ def singlesegment(
     n_manning_cc=None,  # ncc
     cs=None,  # cs
     s0=None,  # s0
-    velp=None,  # velocity at previous time step
     depthp=None,  # depth at previous time step
+    velp=None,  # DUMMY -- dropped from the computation
```

Comment:
OK! I was initially worried that the velp variable would be one too many, but it is hardwired to NONE - so there should be no problems. I am going to test the mc-wave-routing-demo in Colab from this branch to be sure.

Reply:
Notebook will indeed run just fine. Confirmed with test in Colab from this branch

Reply:
Noted. Thanks.